Simplenote.xcodeproj/project.pbxproj

```diff
@@ -4659,7 +4659,7 @@
 				CLANG_ENABLE_OBJC_WEAK = YES;
 				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
-				CODE_SIGN_ENTITLEMENTS = SimplenoteWidgetsExtensionDebug.entitlements;
+				CODE_SIGN_ENTITLEMENTS = Simplenote/SimplenoteWidgetsExtensionDebug.entitlements;
```

Comment:
I got this error [in CI](https://app.circleci.com/pipelines/github/Automattic/simplenote-ios/3838/workflows/199eebfe-d403-4ca1-a651-8496d8998ecf/jobs/4113):

<img width="2475" alt="Screen Shot 2021-06-03 at 12 41 41 pm" src="https://user-images.githubusercontent.com/1218433/120578427-24074b00-c469-11eb-962e-9738a501fd41.png">

Matched by this one locally:

<img width="796" alt="Screen Shot 2021-06-03 at 12 37 58 pm" src="https://user-images.githubusercontent.com/1218433/120578468-33869400-c469-11eb-9623-28bb11e08f83.png">